packages/ui/src/app/pages/Council/Council.tsx

```diff
@@ -25,11 +29,22 @@ export const Council = () => {
 
   const main = (
     <MainPanel ref={sideNeighborRef}>
-      <CouncilList councilors={councilors} onSort={() => undefined} isLoading={isLoading} />
+      {!isLoading && councilors.length === 0 ? (
+        <NotFoundText>There is no council member at the moment</NotFoundText>
+      ) : (
+        <CouncilList councilors={councilors} order={order} onSort={setOrder} isLoading={isLoading} />
+      )}
     </MainPanel>
   )
 
   const sidebar = <SidePanel neighbor={sideNeighborRef} />
 
   return <PageLayout header={header} main={main} sidebar={sidebar} />
 }
+
+const sortBy = ({ key, isDescending }: CouncilOrder): ((a: Councilor, b: Councilor) => number) => {
+  const direction = isDescending ? -1 : 1
+  return key === 'member'
+    ? (a, b) => a.member.handle.localeCompare(b.member.handle) * direction
+    : (a, b) => (a[key] - b[key]) * direction
+}
```

Comment:
@jodator in my understanding the current council will have a small number of member, so I'm thinking that sorting the data locally makes more sense than sending queries. WDYT?

Reply:
Yeah, I saw that too. I'm for sorting locally since we would get already the whole council as you wrote.